baselines/agents/simple_agents.py

```diff
@@ -135,20 +129,15 @@ def get_agent_cls(agent_class_name):
 
 def main():
     parser = argparse.ArgumentParser()
-    parser.add_argument(
-        "--task-config", type=str, default="tasks/pointnav.yaml"
-    )
+    parser.add_argument("--success_distance", type=float, default=0.2)
```

Comment:
"--success_distance" -> `--success-distance`. Note that this does not require changes in python code, python argpase automatically maps `--success-distance` to `args.success_distance`.

Reply:
@abhiskk can you give more context why `success-distance` is better than `success_distance`?

Reply:
It is not necessarily better, but generally I have seen `success-distance` type cmd arguments being used in python, I looked around though and I have found both types of examples actually so don't have a preference here. We should be consistent, the only other place I think command line args are being used in the project are for PPO: https://github.com/facebookresearch/habitat-api/blob/master/baselines/rl/ppo/utils.py#L275  and they are in the format of `--success-distance`. I am happy with either usage, but we should be consistent.

Reply:
I tend to think that `--success-distance` looks cleaner, although it makes it a little harder to search for the definition of the command line arg for a certain variable.